packages/node-firebird-driver/src/lib/index.ts

```diff
@@ -1,5 +1,9 @@
+interface WithIsValidFlag {
```

Comment:
Please remove that interface. We don't want to have an inheritance thing in TS just to have a common method.